web/source/osk/oskManager.ts

```diff
@@ -934,6 +928,19 @@ namespace com.keyman.osk {
       return true;
     }
 
+    private setSize(width?: number, height?: number) {
+      if(width && height) {
+        this.width = width;
+        this.height = height;
+      }
+
+      if(this.vkbd) {
+        this.vkbd.kbdDiv.style.width=this.width+'px';
+        this.vkbd.kbdDiv.style.height=this.height+'px';
+        this.vkbd.kbdDiv.style.fontSize=(this.height/8)+'px';
+      }
+    }
+
```

Comment:
Compare with lines 861-864 below:

https://github.com/keymanapp/keyman/blob/f20098244aebc2a8b157c2e2e147d7bfd9e69262/web/source/osk/oskManager.ts#L861-L864

The bug arose because `this.width` and `this.height` were previously neglected by a handler or two.

(From the method quoted below)

https://github.com/keymanapp/keyman/blob/f20098244aebc2a8b157c2e2e147d7bfd9e69262/web/source/osk/oskManager.ts#L847-L867